aidants_connect_web/templates/footer/faq.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'layouts/main.html' %}
+
+{% load static %}
+
+{% block extracss %}
+<link href="{% static 'css/home.css' %}" rel="stylesheet">
+{% endblock extracss %}
+
+{% block content %}
+<section class="section section-white">
+  <div  class="container">
+    <h1 class="section__title">Foire aux questions Aidants Connect</h1>
```

Comment:
Est-il utile de repréciser dans le titre qu'il s'agit de la FAQ "Aidants Connect" ? 
```suggestion
    <h1 class="section__title">Foire aux questions</h1>
```